docs/modules/plugins/pages/plugin-mongodb.adoc

```diff
@@ -1 +1,121 @@
-include::ROOT:page$under-construction.adoc[]
+= MongoDB Plugin
+
+The plugin provides functionality to interact with https://www.mongodb.com/[MongoDB].
+
+== Installation
+
+.build.gradle
+[source,gradle,subs="attributes+"]
+----
+implementation(group: 'org.vividus', name: 'vividus-plugin-mongodb', version: '{current-version}')
+----
+
+== Properties
+
+It's allowed to configure unlimited number of MongoDB connections via mechanism of the dynamic properties. The properties prefix example is:
+```properties
+mongodb.connection.mock=mongodb+srv://username:password@mock-zzw4a.mongodb.net/test?retryWrites=true&w=majority
+```
+where `mock` is a key used to refer MongoDB connection in the steps. The key is defined by users, must be unique and can't contain dots.
+
+== Steps
+
+=== Execute native command
+
+Executes a mongo native command against a mongo database and saves command's result to a variable
+
+[NOTE]
+====
+* https://docs.mongodb.com/manual/reference/command/[Native commands]
```

Comment:
give short description what native commands are

Reply:
renamed to database command as per docs